testing/sdk_marathon.py

```diff
@@ -225,22 +231,29 @@ def _update():
 
 def destroy_app(app_name: str, timeout=TIMEOUT_SECONDS) -> None:
     @retrying.retry(stop_max_delay=timeout * 1000, wait_fixed=2000)
-    def _destroy():
+    def _destroy() -> dict:
         response = sdk_cmd.cluster_request(
             "DELETE", _api_url("apps/{}".format(app_name)), params={"force": "true"}
         )
-        return _handle_marathon_deployment_response(response)
+        return response.json()
```

Comment:
Would it not make sense to rather include the `deploymentId` in the `_deployment_result` class than handling it directly here.

Reply:
The problem is that the `_deployment_result` is also used for `GET`  requests on `apps/{name}` to parse the response. Hereby the `deploymentId` is not included. Mixing both didn't seem like a good idea.

Reply:
Here: https://github.com/mesosphere/dcos-commons/blob/DCOS-42770-wait-until-marathon-app-deletion-finished/testing/sdk_marathon.py#L141

The deploymentID is explicitly mentioned, so it would make sense to include it there too.

Reply:
Yes, it is included in the comment but the key is not in the response json. I think someone had the idea of building a serialization with this class but then discovered the `deploymentId` is not always present.